docs/index.js.html

```diff
@@ -125,25 +122,25 @@ <h1 class="page-title">Source: index.js</h1>
   if (hasBrowserSupport) {
     return sheet;
   }
-  const noSpanningCSS = replaceSpanningMediaBlocks(sheet, "");
-  const spanningCSS = getSpanningCSSText(sheet);
+  const noViewportSegments = replaceViewportSegmentsMediaBlocks(sheet, "");
+  const viewportSegmentCSS = getViewportSegmentCSSText(sheet);
 
   if (elementName) {
-    spanning[elementName] = {
-      "single-fold-horizontal": "",
-      "single-fold-vertical": "",
-      "none": ""
-    };
+    viewportSegments[elementName] = [[]];
   }
 
-  const _spanning = elementName ? spanning[elementName] : spanning;
+  const _viewportSegments = elementName ? viewportSegments[elementName] : viewportSegments;
   const sheetOrigin = elementName ? '' : "/* origin: inline */";
-  for (let [key, value] of Object.entries(spanningCSS)) {
-    _spanning[key] += `${sheetOrigin}${value}`;
+  for (let [key, value] of Object.entries(viewportSegmentCSS)) {
+    if (_viewportSegments[key] == undefined)
+    _viewportSegments[key]= [];
```

Comment:
any reason why we are doing these _names, it is a ES module right so not exposed